### ISSUE TITLE
cointracker format example

### ISSUE THREAD

**Issue by berlinguyinca** [eprbell/dali-rp2#232]
good afternoon, would it be possible to get an input example for coin tracker files?

They look like this:

Date,Type,Transaction ID,Received Quantity,Received Currency,Received Cost Basis (USD),Received Wallet,Received Address,Received Comment,Sent Quantity,Sent Currency,Sent Cost Basis (USD),Sent Wallet,Sent Address,Sent Comment,Fee Amount,Fee Currency,Fee Cost Basis (USD),Realized Return (USD),Fee Realized Return (USD)
4/6/2024 21:19,LOST,018eb548-820b-780e-9d85-4259d3d49099,,,...,,,,155957.195,VET,...,Voyager,,lost,,,...,...,...
4/6/2024 16:46,SEND,018eb534-c022-7d14-a29d-3b8b0374602c,,,...,,,,76,USD,...,Coinbase USD Wallet,,"Withdrew funds
To BANK OF AMERICA, N.A. ************",,,...,...,...
4/6/2024 14:56,RECEIVE,018eb534-a0fc-7432-b797-2cd98c781fe0,1,NFT #1,...,Polygon Wallet ...EE57Cd,0xD5b2308908C96466f27C8aD286736028A9EE57Cd,,,,...,,,,,,...,...,...
4/6/2024 2:51,SEND,018eb534-c01f-702a-aee4-eadc6d50fa56,,,...,,,,29.96,USD,...,Coinbase USD Wallet,,"Spent US Dollar (authorization)
Using Coinbase Card",,,...,...,...
4/5/2024 23:11,RECEIVE,018eb534-c01c-7fd2-a24c-a826f9991673,140,USD,...,Coinbase USD Wallet,,"Deposited funds
From BANK OF AMERICA, N.A. ************",,,...,,,,,,...,...,...
4/5/2024 10:46,STAKING_REWARD,018eb534-c040-7bbb-a786-26e64fc6dd28,0.001903,XTZ,...,Coinbase XTZ Wallet,,"Tezos reward
From Coinbase",,,...,,,,,,...,...,...
4/5/2024 0:49,INTEREST_PAYMENT,018eb534-c031-7435-8599-5146b0802d73,0.039315,USDC,...,Coinbase USDC Wallet,,"USDC reward
From Coinbase",,,...,,,,,,...,...,...
4/4/2024 18:45,STAKING_REWARD,018eb534-c06c-72c8-b0e1-4d4d904a6fd6,0.0000001,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,...
4/4/2024 15:29,STAKING_REWARD,018eb534-c069-7c31-a471-feb528d35a3d,0.00000182,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,...
4/3/2024 19:22,RECEIVE,018eb534-a0f3-7ad1-b08d-65d0894f1b98,100,WGC,...,Polygon Wallet ...EE57Cd,0xD5b2308908C96466f27C8aD286736028A9EE57Cd,,,,...,,,,,,...,...,...
4/2/2024 13:50,STAKING_REWARD,018eb534-c03d-7131-befd-041877587754,0.001783,XTZ,...,Coinbase XTZ Wallet,,"Tezos reward
From Coinbase",,,...,,,,,,...,...,...
4/2/2024 11:18,STAKING_REWARD,018eb534-c066-77c1-95d5-035c922186c8,0.00000009,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,...
4/1/2024 21:02,STAKING_REWARD,018eb534-c063-7739-a4e1-b1a55596a68f,0.00000165,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,...
4/1/2024 15:44,RECEIVE,018eb53f-b6a2-795d-8cb2-6c6f83fd25cc,50,SCAM,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,,,...,,,,,,...,...,...
4/1/2024 5:49,RECEIVE,018eb53f-b69f-7249-ba77-205dc1fc2fec,384000,CWIF,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,,,...,,,,,,...,...,...
3/31/2024 12:47,RECEIVE,018eb534-d0c5-7eb9-99eb-cc1325344378,783158000.00000000,CHICKENCOIN,...,Ethereum Wallet ...EE57Cd,0xD5b2308908C96466f27C8aD286736028A9EE57Cd,,,,...,,,,,,...,...,...
3/30/2024 23:30,STAKING_REWARD,018eb534-c060-7dc5-853c-3c7ae077cfdb,0.00000012,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,...
3/30/2024 22:48,TRADE,018eb53f-b698-7c51-b6c3-d49e250f28ae,1165.220261,MYRO,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,2,SOL,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,0.000105,SOL,...,...,...
3/30/2024 22:48,TRADE,018eb53f-b698-7c51-b6c3-d49e250f28ae,,,...,,,,0.00203928,SOL,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,,,...,...,...
3/30/2024 21:55,TRADE,018eb53f-b691-789e-b52f-ffc7be154395,34904.55151,HUMP,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,1,SOL,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,0.000105,SOL,...,...,...
3/30/2024 21:55,TRADE,018eb53f-b691-789e-b52f-ffc7be154395,,,...,,,,0.00203928,SOL,...,Solana Wallet ...P1LX35,DThwDDCsrniDXRt6rv7KmJJCMfA88BuXr9RaiCP1LX35,,,,...,...,...
3/30/2024 18:39,STAKING_REWARD,018eb534-c05d-7f64-9f21-0f6b4040f3c5,0.00000164,SOL,...,Coinbase SOL Wallet,,"Solana reward
From Coinbase",,,...,,,,,,...,...,..

thank you and appreciate it

**Comment by macanudo527** [comment]
Did you mean to post your personalized data?  I deleted it just in case. If you are okay with sharing your personalized data, you can recover the edit. I wouldn't be.

If you do not want to share your personalized data, be sure to also delete the edit that contains it. I can not do that.

What would be helpful is if you uploaded a CSV with sanitized data (things like transaction ids removed and bank names changed etc...)

**Comment by berlinguyinca** [comment]
yes I meant to post these, but I can make a sanitized version here in a bit

**Comment by macanudo527** [comment]
> yes I meant to post these, but I can make a sanitized version here in a bit

It's up to you. I just wanted to make sure you knew that your private information would be available on the internet for anyone to do with as they please.

**Comment by macanudo527** [comment]
BTW I'm pretty sure the [Pionex plugin](https://github.com/eprbell/dali-rp2/blob/main/src/dali/plugin/input/csv/pionex.py) uses a file that is similar to the cointracker format. You might want to give it a try and see if it works for you.
